jobs/desktop-mobile-mau-2020/mobile_mau/forecast_and_actual.sql

```diff
@@ -0,0 +1,121 @@
+WITH forecast_base AS (
+  SELECT
+    REPLACE(datasource, " Global MAU", "") AS datasource,
+    date,
+    type,
+    value AS mau,
+    low90,
+    high90,
+    p10 AS low80,
+    p90 AS high80,
+    p20,
+    p30,
+    p40,
+    p50,
+    p60,
+    p70,
+    p80
+  FROM
+    `moz-fx-data-shared-prod.telemetry.simpleprophet_forecasts`
+  WHERE
+    asofdate = (
+      SELECT
+        MAX(asofdate)
+      FROM
+        `moz-fx-data-shared-prod.telemetry.simpleprophet_forecasts`
+    )
+    AND datasource IN (
+      "Fenix Global MAU",
+      "Fennec Global MAU",
+      "Firefox iOS Global MAU",
+      "Firefox Lite Global MAU",
+      "Firefox Echo Global MAU",
+      "Focus Android Global MAU",
+      "Focus iOS Global MAU",
+      "Lockwise Android Global MAU"
+    )
+),
+mobile_base AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.telemetry.firefox_nondesktop_exact_mau28_by_dimensions_v1`
+  WHERE
+    product IN (
+      "Fenix",
+      "Fennec",
+      "Firefox iOS",
+      "Firefox Lite",
+      "Firefox Echo",
+      "Focus Android",
+      "Focus iOS",
+      "Lockwise Android"
+  )
+),
+per_bucket AS (
+  SELECT
+    product AS datasource,
+    'actual' AS type,
+    submission_date,
+    id_bucket,
+    SUM(mau) AS mau
+  FROM
+    mobile_base
+  GROUP BY
+    product,
+    id_bucket,
+    submission_date
+),
+with_ci AS (
+  SELECT
+    datasource,
+    type,
+    submission_date,
+    `moz-fx-data-shared-prod.udf_js.jackknife_sum_ci`(20,
+      ARRAY_AGG(mau)) AS mau
```

Comment:
Have you passed this through the `format_sql` script in bigquery-etl, by any chance?

Reply:
No but I guess I might as well